.travis.yml

```diff
@@ -7,16 +7,15 @@ cache:
     - $HOME/.composer/cache/files
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
   - 5.6
   - hhvm
```

Comment:
I suggest removing the HHVM job, given that HHVM does not care about PHP compatibility anymore: http://hhvm.com/blog/2017/09/18/the-future-of-hhvm.html